### ISSUE TITLE
Tensorflow - Summary

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#22]
# Reference
- [ ] [Tensorflow Tutorials](https://tensorflow.google.cn/tutorials/)
- [x] [Youtube 炼数成金 tensorflow 课程](https://www.youtube.com/watch?v=eAtGqz8ytOI&list=PLjSwXXbVlK6IHzhLOMpwHHLjYmINRstrk&index=1)
- [ ] [Stanford University class - CS20SI Tensorlfow](https://www.youtube.com/watch?v=g-EvyKpZjmQ&index=1&list=PLIDllPt3EQZoS8gCP3cw273Cq9puuPLTg)
- [ ] [tensorflow 资源](https://www.cnblogs.com/stoker/p/9101820.html)
- [ ] [tensorflow 源码分析](http://www.cnblogs.com/jicanghai/tag/tensorflow/)
- [ ] [datasets](https://github.com/junxnone/junxnone.github.io/issues/30)
- [ ] [Google ML course](https://developers.google.com/machine-learning/crash-course/)
- [ ] [tensorflow - gpu - cuda -cudnn - gcc version](https://www.tensorflow.org/install/source#gpu)

# Brief
- TensorBoard
- TFRecord
- tensorflow.keras
- tfjs
- slim

##  History

# Tensorflow official tutorials
- [ ] [Get Started with TensorFlow](https://tensorflow.google.cn/tutorials/)
- [ ] [Learn and use machine learning](https://tensorflow.google.cn/tutorials/keras/)
  - [x] [Train your first neural network: basic classification](https://tensorflow.google.cn/tutorials/keras/basic_classification)
  - [x] [Text classification with movie reviews](https://tensorflow.google.cn/tutorials/keras/basic_text_classification)
  - [x] [Predict house prices: regression](https://tensorflow.google.cn/tutorials/keras/basic_regression)
  - [x] [Explore overfitting and underfitting](https://tensorflow.google.cn/tutorials/keras/overfit_and_underfit)
  - [x] [Save and restore models](https://tensorflow.google.cn/tutorials/keras/save_and_restore_models)
- [ ] [Research and experimentation](https://tensorflow.google.cn/tutorials/eager/)
- [ ] [Learn more about TensorFlow](https://tensorflow.google.cn/tutorials/next_steps)

# Colab examples
- [x] [basic_classification - Fashion MNIST](https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/keras/basic_classification.ipynb#scrollTo=gut8A_7rCaW6)
- [x] [basic-text-classification - Text classification with movie reviews](https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/keras/basic_text_classification.ipynb)
- [x] [basic-regression - Predict house prices: regression](https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/keras/basic_regression.ipynb#scrollTo=EIdT9iu_Z4Rb)

#  Log 
```
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
```
num | 输出信息
-- | --
“0” | INFO + WARNING + ERROR + FATAL
“1” | WARNING + ERROR + FATAL
“2” | ERROR + FATAL
“3” |	FATAL

**Comment by junxnone** [comment]
loss: mse 
- mean squared error  

metrics: mae 
- mean absolute error

**Comment by junxnone** [comment]
[tf.expand_dims()](https://www.tensorflow.org/api_docs/python/tf/expand_dims)
Inserts a dimension of 1 into a tensor's shape. (deprecated arguments)

[tf.image.resize_bilinear](https://www.tensorflow.org/versions/r1.0/api_docs/python/tf/image/resize_bilinear)
Resize images to size using bilinear interpolation.

**Comment by junxnone** [comment]
#5 

# TODO
- [ ] [PoseNet & Move Mirror](https://experiments.withgoogle.com/collection/ai/move-mirror/view)
- [ ] [dopamine](https://github.com/google/dopamine)
- [ ] [Machine Learning Fairness](https://developers.google.cn/machine-learning/fairness-overview/)
